Packs/jamf/Integrations/jamfV2/jamfV2.py

```diff
@@ -28,6 +29,73 @@
 
 
 class Client(BaseClient):
+    def __init__(self, base_url: str, verify: bool, username: str = "", password: str = "", proxy: bool = False):
+        super().__init__(base_url=base_url, auth=(username, password), verify=verify, proxy=proxy)
+
+        """ due to deprecating the basic auth option from the classical API versions 10.35 and up
+            the client will try to generate an auth token first, if it failed to do generate the token,
+            the client will use basic auth instead.
```

Comment:
For some reason, I don't see the implementation for the fallback from token to base creds, where is it?

Reply:
We are using the basic auth to get the token, if getting the token failed we will just continue using the basic auth instead